lib/compat.php

```diff
@@ -44,6 +44,9 @@ function gutenberg_safe_style_css_column_flex_basis( $attr ) {
  */
 function gutenberg_provide_render_callback_with_block_object( $pre_render, $block ) {
 	global $post;
+	if ( 'core/navigation' !== $block['blockName'] ) {
+		return null;
```

Comment:
It makes no effective difference, but if the filter operates on the assumption that anything other than the default value being returned should be interpreted as bypassing `render_block`, then we should just pass back that default.

i.e.

```suggestion
		return $pre_render;
```

Actually, it occurs to me that maybe this isn't effectively the same, and ought to be changed, because otherwise it prevents any other plugins which add their filters at an earlier priority from being able to bypass `render_block` as they're expecting to be able to.

Returning `$pre_render` would support those cases, in addition to the default `null` passed through `render_block`.

Reply:
Good catch! Thanks!